README.md

```diff
@@ -193,22 +193,40 @@ If you have raster data, first convert it to a shapefile. See [Converting raster
 
 ### Fully automated pipeline
 
-If the structure of your text content is simple enough, you can import shapefiles and snuggets automatically without having to do much manual work. We recommend using this pathway if possible, because it makes moving the site to a new server significantly easier. To do this, you will need a `snuggets.xlsx` file with the same columns as the example one we've included in `data.zip`. The columns can be in any order, but the headings must be exactly as typed here:
+If the structure of your text content is simple enough, you can import shapefiles and snuggets automatically without having to do much manual work. We recommend using this pathway if possible, because it makes moving the site to a new server significantly easier. To do this, you will need a `snuggets.csv` file with the same columns as the example one we've included in `data.zip`. The columns can be in any order, but the headings must be exactly as typed here:
```

Comment:
For Seattle I switched the snugget loader to use Excel files.  I think that's worth backporting into the main project, because it'll make handling translations much easier.  Long story short, CSVs saved from Excel garble non-ASCII characters, and in my experience people manually making or editing CSV files are almost always using Excel to do so.

I also see that in switching this to .csv here you're making the readme consistent with the version of `snugget_load.py` in this repository, so I think this change makes sense for now.  I will put a task on my list to switch the snugget loader over, outside the scope of this PR.